zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -117,6 +115,53 @@ where
     Ok(())
 }
 
+pub type ProposeTransferErrT<DbT, CommitmentTreeErrT, InputsT, ChangeT> = Error<
```

Comment:
A short bit of documentation for each of these new type aliases would be useful (even just a single sentence linking to the method for which this error is used).